generated/google_cloud_protobuf/lib/src/encoding.dart

```diff
@@ -155,3 +155,39 @@ Map<T, Object?>? encodeMap<T>(Map<T, JsonEncodable>? value) =>
 /// Encode a list of `bytes` values into JSON.
 Map<T, String>? encodeMapBytes<T>(Map<T, Uint8List>? value) =>
     value?.map((key, value) => MapEntry(key, base64Encode(value)));
+
+/// Extensions methods used for comparing to proto default values.
+extension BoolExtension on bool {
```

Comment:
Maybe these should be something like `*ProtoDefault` since they will appear in generated documentation.